hammercloud/hammercloud.py

```diff
@@ -99,7 +99,7 @@ def main():
             return
         if ret:
             subprocess.call('{0}/{1}.sh'.format(hammercloud.cachedir, ret), shell=True)
-    elif isinstance(servers, list):
+    elif isinstance(servers, list) and servers:
```

Comment:
change this, put if not servers above the whole statement, and just return in it, so it skips the above if statement as well